Octokit.Tests/Clients/UserGpgKeysClientTests.cs

```diff
@@ -0,0 +1,119 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using NSubstitute;
+using Xunit;
+
+namespace Octokit.Tests.Clients
+{
+    public class UserGpgKeysClientTests
+    {
+
+        public class TheCtor
+        {
+            [Fact]
+            public void EnsuresNonNullArguments()
+            {
+                Assert.Throws<ArgumentNullException>(() => new UserGpgKeysClient(null));
+            }
+        }
+
+        public class TheGetAllForCurrentMethod
+        {
+            [Fact]
+            public void RequestsTheCorrectUrl()
+            {
+                var connection = Substitute.For<IApiConnection>();
+                var client = new UserGpgKeysClient(connection);
+
+                var expectedUri = ApiUrls.GpgKeys().ToString();
+                client.GetAllForCurrent();
+
+                connection.Received().GetAll<GpgKey>(
+                    Arg.Is<Uri>(u => u.ToString() == expectedUri),
+                    Arg.Any<Dictionary<string, string>>(),
+                    Arg.Is<string>(s => s == AcceptHeaders.GpgKeysPreview),
```

Comment:
Similarly, we hardcode the accept header value on unit tests so we dont have the unit test and implementation code both relying on the `AcceptHeaders` helper class